### DISCUSSION TITLE
Hello, my name Yevgen from Ukraine. I'm a beginner in Full Stack Development, looking for a mentor.

### DISCUSSION THREAD

**Discussion by yevgenperedkov** [community/community#107543] [New to GitHub]
Hello, my name Yevgen from Ukraine. I'm a beginner in Full Stack Development, looking for a mentor.
If there any person who can dedicate some time to learn me, or give some non-payable job to make some experience.
I would like to move to LA in near future with my family (wife and daughter) for a better life, I hope.
Will appreciate every kind of help. Thank you for your respond.
Looking forward to hear from somebody.

P.S.: I'm a fast learner, non-problematic, ready to work/learn hard. More tuned in to find a job in some US (LA) company as a beginner with no profit. To make portfolio and applicate in a year for a job and move there. Again, thank you so much.

Best regards,
Yevgen

**Comment by jane-does-coding** [comment]
I can't teach you, however, I would reccomend setting up your profile readme file, to show people what programming skills and languages you knnow so it will be easier for people to understand what kind of a developer you are and what you code in. If you are intrested I can reccommend good courses and videos that are project-based learning.
